core/api/src/migrations/000036-createExportRuns.ts

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  up: async function (migration, DataTypes) {
+    await migration.createTable("exportRuns", {
+      guid: {
+        type: DataTypes.STRING(40),
+        primaryKey: true,
+      },
+
+      exportGuid: {
+        type: DataTypes.STRING(40),
+        allowNull: false,
+      },
+
+      runGuid: {
+        type: DataTypes.STRING(40),
+        allowNull: false,
+      },
+
+      createdAt: {
+        type: DataTypes.DATE,
+        allowNull: false,
+      },
+
+      updatedAt: {
+        type: DataTypes.DATE,
+        allowNull: false,
+      },
+    });
+
+    await migration.addIndex("exportRuns", ["exportGuid", "runGuid"], {
+      unique: true,
+      fields: ["exportGuid", "runGuid"],
+    });
+
+    await migration.removeColumn("exports", "runGuids");
```

Comment:
! This is a destructive migration, but the export data isn't too important to keep around...